backend-api/src/functions/asyncTxmaEvent/asyncTxmaEventHandler.test.ts

```diff
@@ -0,0 +1,93 @@
+import { expect } from "@jest/globals";
+import { APIGatewayProxyResult, Context } from "aws-lambda";
+import "dotenv/config";
+import "../../../tests/testUtils/matchers";
+import { logger } from "../common/logging/logger";
+import { buildLambdaContext } from "../testUtils/mockContext";
+import { buildRequest } from "../testUtils/mockRequest";
+import {
+  expectedSecurityHeaders,
+  mockSessionId,
+} from "../testUtils/unitTestData";
+import { lambdaHandlerConstructor } from "./asyncTxmaEventHandler";
+import { IAsyncTxmaEventDependencies } from "./handlerDependencies";
+
+jest.mock("crypto", () => ({
+  ...jest.requireActual("crypto"),
+  randomUUID: () => "mock_opaque_id",
+}));
+
+describe("Async TxMA Event", () => {
+  let dependencies: IAsyncTxmaEventDependencies;
+  let context: Context;
+  let consoleInfoSpy: jest.SpyInstance;
+  let result: APIGatewayProxyResult;
+
+  const validRequest = buildRequest({
+    body: JSON.stringify({
+      sessionId: mockSessionId,
+      eventName: "DCMAW_READID_NFC_BILLING_STARTED",
+    }),
+  });
```

Comment:
nitpick (non-blocking): I don't think the body override is needed and should come in the next PR

```suggestion
  const validRequest = buildRequest();
```